### ISSUE TITLE
[Hyper-parameters] Training script

### ISSUE THREAD

**Issue by ElectronicElephant** [westerndigitalcorporation/YOLOv3-in-PyTorch#8]
Hello, thank you for your awesome work!

I'm wondering the meaning of **Retrained weight PyTorch** in `README.md`.

Did you train the whole network from scratch, or, just load the darknet weights and finetune the last few layers?

Can you share the training hyper-parameters with me? Are they just the default values in `main.py`?

**Comment by ElectronicElephant** [comment]
Also, I don't know the normal range of the loss functions. Could you please share the training log with us?

**Comment by eng100200** [comment]
@ElectronicElephant yes, i am alo wondering about the loss function and the normal range, since the total loss is sum and its value is mostly in 10--1000.

**Comment by ElectronicElephant** [comment]
> @ElectronicElephant yes, i am alo wondering about the loss function and the normal range, since the total loss is sum and its value is mostly in 10--1000.

Hi, the author and I are trying to merge it into mm-detection, which is a more complete and capable framework. You can refer to the pr there. It's expected to be merged by the end of July.

**Comment by eng100200** [comment]
@ElectronicElephant  did you get correct results after training? please reply

**Comment by ElectronicElephant** [comment]
> @ElectronicElephant  did you get correct results after training? please reply

No, not this repo. I failed to train this repo.

But with the one in mm-detection, everything works like a charm.

**Comment by eng100200** [comment]
@ElectronicElephant I knew this repo can not be trained, it has problem in loss function , not working right way. I have never used MM-detection? Do you share how to start with MM-detection?

**Comment by ElectronicElephant** [comment]
> @ElectronicElephant I knew this repo can not be trained, it has problem in loss function , not working right way. I have never used MM-detection? Do you share how to start with MM-detection?

Well, I think Mm-detection has very detailed documents. You can just check that.
